app/handlers/base.py

```diff
@@ -20,6 +24,51 @@
 env, cfg = load_env()
 
 
+# Monkey-patch Python Social Auth's base handler
+#
+# See
+# https://github.com/python-social-auth/social-app-tornado/blob/master/social_tornado/handlers.py
+# for the original
+#
+# Python Social Auth documentation:
+# https://python-social-auth.readthedocs.io/en/latest/backends/implementation.html#auth-apis
+
+class PSABaseHandler(RequestHandler):
+    """
+    Mixin used by Python Social Auth
+    """
+    def user_id(self):
+        return self.get_secure_cookie('user_id')
+
+    def get_current_user(self):
+        user_id = self.user_id()
+        oauth_uid = self.get_secure_cookie('user_oauth_uid')
```

Comment:
Why define a method for `user_id` but not `user_oauth_uid`?

Reply:
That method is defined in PSA, so I just kept it as is since I wasn't sure if they used it anywhere.

Do you want me to remove it?